src/styles/Learn.module.css

```diff
@@ -20,7 +20,7 @@
   text-align: left;
 }
 .imageWrapper {
-  text-align: center;
+  margin-left: 84px;
```

Comment:
Since we're trying to move away from CSS modules, it would be better to put this directly in the learn page using `Box` and `sx prop`

Reply:
Done @paulcretu